api/src/pcapi/core/offers/repository.py

```diff
@@ -633,3 +633,8 @@ def get_synchronized_offers_with_provider_for_venue(venue_id: int, provider_id:
     return Offer.query.filter(Offer.venueId == venue_id).filter(
         Offer.lastProviderId == provider_id  # pylint: disable=comparison-with-callable
     )
+
+
+def update_stock_quantity_to_dn_booked_quantity(stock: Stock) -> None:
+    Stock.query.filter(Stock.id == stock.id).update({"quantity": stock.dnBookedQuantity})
+    db.session.commit()
```

Comment:
Ici il faut noter la différence de majuscule + du paramètre en entrée:
```suggestion
def update_stock_quantity_to_dn_booked_quantity(stockId: int) -> None:
    Stock.query.filter(Stock.id == stockId).update({"quantity": Stock.dnBookedQuantity})
    db.session.commit()
```

Reply:
Ça fonctionne pour le cas d'usage

Reply:
étrange ca ne fonctionne pas j'ai l'erreur suivante
_update({"quantity": Stock.dnBookedQuantity})_ le Stock.dnBookedQuantity n'a pas l'air d'être une valeur utilisable
 
```
/home/pcapi/.local/lib/python3.10/site-packages/sqlalchemy/sql/sqltypes.py:3333: in _resolve_value_to_type
    raise exc.ArgumentError(
E   sqlalchemy.exc.ArgumentError: Object <Stock #1/AE> is not legal as a SQL literal value

The above exception was the direct cause of the following exception:
tests/core/offers/test_repository.py:1329: in test_update_stock_quantity_to_dn_booked_quantity
    repository.update_stock_quantity_to_dn_booked_quantity(stock)
src/pcapi/core/offers/repository.py:639: in update_stock_quantity_to_dn_booked_quantity
    Stock.query.filter(Stock.id == stock_id).update({"quantity": Stock.dnBookedQuantity})
```